sklearn_numba_dpex/common/kernels.py

```diff
@@ -102,7 +109,7 @@ def initialize_to_zeros(data):
 
 
 @lru_cache
-def make_broadcast_division_1d_2d_axis0_kernel(size0, size1, work_group_size):
+def make_broadcast_division_1d_2d_axis0_kernel(size0, size1, work_group_size, dtype):
```

Comment:
Is passing the dtype requiring here as well? There is no occurrence of `dpex.func` in this function, only a `dpex.kernel`.

Or maybe we can keep it so as to get a consistent signature style for all kernel factory function, irrespective of the fact that they use device functions or not.

Reply:
You're right it's a mistake, thanks for pointing it out.